coretex/entities/task_run/parameter/parameters/range_parameter.py

```diff
@@ -0,0 +1,74 @@
+#     Copyright (C) 2023  Coretex LLC
+
+#     This file is part of Coretex.ai
+
+#     This program is free software: you can redistribute it and/or modify
+#     it under the terms of the GNU Affero General Public License as
+#     published by the Free Software Foundation, either version 3 of the
+#     License, or (at your option) any later version.
+
+#     This program is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU Affero General Public License for more details.
+
+#     You should have received a copy of the GNU Affero General Public License
+#     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from typing import Any, List, Dict, Optional, Tuple
+
+from ..base_parameter import BaseParameter
+from ..utils import validateRangeStructure
+from ....project import ProjectType
+
+
+class RangeParameter(BaseParameter[Dict[str, Any]]):
```

Comment:
Since we know the value of `range` parameter keys we can use them here instead of `Any`